test/JLM_Tests/playlist.xml

```diff
@@ -1061,6 +1061,35 @@
 		</subsets>
 	</test>
 
+	<test>
+		<testCaseName>testOpenJ9DiagnosticsMXBean</testCaseName>
+		<variations>
+			<variation>Mode100</variation>
+			<variation>Mode101</variation>
+		</variations>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) \
+	$(MKTREE) $(REPORTDIR); \
+	cd $(REPORTDIR); \
+	$(JAVA_COMMAND) $(JVM_OPTIONS) \
+	-Xdump:dynamic \
+	-cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)jlm_tests.jar$(Q) \
+	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) \
+	-testnames testOpenJ9DiagnosticsMXBean \
+	-groups $(TEST_GROUP) \
+	-excludegroups $(DEFAULT_EXCLUDE); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<subsets>
+			<subset>SE80</subset>
+			<subset>SE90</subset>
```

Comment:
This test should apply to all Java versions: can we just remove the `<subsets>` element?

Reply:
No, in able to support special subsets like Panama and Valhalla, we are caught in the position of explicitly stating the subset.  If you do remove < subsets >, it will generate make targets for 5-6 supported subsets (SE80, SE90, SE100, SE110, Panama, Valhalla) and test compilation will fail, as Panama and Valhalla have special requirements for build.  (At some point we should address this in a more satisfying way, in order not to play this game of continuously updating these... perhaps Panama & Valhalla off in a separate branch of openj9? but currently explicit statement is the way)

So, in this case, you may want to add SE100 to run this test also against Java10.